App/Utilities/Sources/Utilities/Design/Typography/DesignSystem.Typography.swift

```diff
@@ -7,17 +7,17 @@ extension DesignSystem {
     public enum Typography {
 
         /// Normal text and buttons.
-        case label(weight: Font.Weight = .regular)
+        case label(weight: Font.Weight = .regular, size: CGFloat = 17)
```

Comment:
Cool idea :)

Reply:
I had something similar for `Padding` and I'm not sure if using this feels right :) There is a temptation to put whatever you want. Also, that breaks the idea of having values you do not control (hidden by the API).

I know sometimes we need to set some custom values, but I added a custom case for padding and I did not used the default values :) We can definitely talk this out :)